srcpkgs/nagios/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'nagios'
+pkgname=nagios
+version=4.4.6
+revision=1
+build_style=gnu-configure
+
+_nagios_user="_nagios"
+_nagios_group="_nagios"
+_instdir="/usr/share/nagios"
+_bindir="/usr/bin"
+_vardir="/var/nagios"
+_confdir="/etc/nagios"
+_httpdconfdir="/etc/webapps/$pkgname/"
+_checkresultdir="/var/nagios/spool/checkresults"
+_cgibindir="/usr/libexec/nagios/cgi"
+
+configure_args="
+	--with-nagios-user=$_nagios_user
+	--with-nagios-group=$_nagios_group
+	--prefix=$_instdir
+	--bindir=$_bindir
+	--localstatedir=$_vardir
+	--sysconfdir=$_confdir
+	--with-httpd-conf=$_httpdconfdir
+	--with-checkresultdir=$_checkresultdir
+	--with-cgibindir=$_cgibindir
+	--enable-embedded-perl"
+make_build_args="all"
+make_install_args="
+	prefix=$_instdir
```

Comment:
Why `prefix` to `/usr/share/....`

Reply:
I based this on the Arch Linux AUR (https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=nagios)

I'll look into this - now that you've pointed it out it doesn't look proper.

Reply:
This is outdated and has been removed in my latest commit from 10 days ago